src/bodypart.h

```diff
@@ -173,6 +173,14 @@ class body_part_set
             return parts.size();
         }
 
+        const cata::flat_set<bodypart_str_id>::iterator begin() const {
+            return parts.begin();
+        }
+
+        const cata::flat_set<bodypart_str_id>::iterator end() const {
```

Comment:
```suggestion
        cata::flat_set<bodypart_str_id>::iterator end() const {
```
Returning const **values** is almost never useful.